app/static/app/js/main.jsx

```diff
@@ -3,5 +3,17 @@ import './django/csrf';
 import React from 'react';
 import ReactDOM from 'react-dom';
 import Dashboard from './Dashboard';
+import Console from './Console';
+import $ from 'jquery';
 
-ReactDOM.render(<Dashboard/>, document.getElementById('dashboard-app'));
+$("[data-dashboard]").each(function(){
```

Comment:
Yes, correct! Since some parts of the app are powered by Django, certain parts are React apps, other parts use the Django forms. So far I think I'll have one app for the dashboard, one for the map display and one for the 3d model display. (Or maybe just a single app that does both map and 3d display).

Since I'm using jQuery this way, perhaps I'll leave `fetch` out of the picture for now.